sas/sas/steps/steps.py

```diff
@@ -96,14 +101,22 @@ def login_user(step, email, password):
 @step(r'I run loaddata to populate dropdowns')
 def run_command_line(step):
     call_command('loaddata', 'buildings', 'places')
+    call_command('loaddata', 'user/fixtures/group.json')
     call_command('loaddata', 'user/fixtures/users.json')
     call_command('loaddata', 'user/fixtures/userProfiles.json')
     call_command('loaddata', 'booking/fixtures/bookTimes.json')
     call_command('loaddata', 'booking/fixtures/bookings.json')
 
+@step(r'this user with email "(.*)" is an admin')
```

Comment:
Porque vcs não criaram uma step só com o texto I register the admin "email" with password ....